iset.mm

```diff
@@ -64111,6 +64111,22 @@ would need to be strengthened (for example, to imply included middle) to
     c0 syl5eqel ) ACEZBDEZABFZGABHAIZBIZJZKABLUAUDKEUBUEKEUCUDUEMSNUFKEACOBDOAB
     PUDUEQRT $.
 
+  ${
+    tpfidisj.a $e |- ( ph -> A e. V ) $.
+    tpfidisj.b $e |- ( ph -> B e. W ) $.
+    tpfidisj.c $e |- ( ph -> C e. X ) $.
+    tpfidisj.ab $e |- ( ph -> A =/= B ) $.
+    tpfidisj.ac $e |- ( ph -> A =/= C ) $.
+    tpfidisj.bc $e |- ( ph -> B =/= C ) $.
+    $( A triple is finite if it consists of three unequal sets.  (Contributed
+       by Jim Kingdon, 1-Oct-2022.) $)
+    tpfidisj $p |- ( ph -> { A , B , C } e. Fin ) $=
```

Comment:
I see that prfidisj's comment has links to the proper class cases. Can you similarly add to snfig's comment a link to snprc ?